src/pages/Book/BookARsv.js

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import styled from 'styled-components';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
+import { useState } from 'react';
+
+const BookARsv = () => {
+  const [value, setValue] = useState(new Date());
+
+  const changeDate = e => {
+    setValue(e);
+  };
+
+  return (
+    <>
+      <ReservationContainer>
+        <PlaceSelect>
+          <Step1>STEP1</Step1>
```

Comment:
```suggestion
          <StepTitle>STEP1</StepTitle>
```
STEP이 보여지는 부분은 동일한 스타일 컴포넌트로 통일하는게 좋을 것 같네요!